src/pages/Overview/AnalyticsBanner.tsx

```diff
@@ -1,23 +0,0 @@
-import { FC } from "react"
-import { useColorModeValue } from "@threshold-network/components"
-import analyticsImageLight from "../../static/images/AnalyticsIllustration.png"
-import analyticsImageDark from "../../static/images/AnalyticsIllustrationDark.png"
-import { ModalType } from "../../enums"
-import { useModal } from "../../hooks/useModal"
-import SecondaryAnnouncementBanner from "../../components/AnnouncementBanner/SecondaryAnnouncementBanner"
```

Comment:
Looks like `SecondaryAnnoucementBanner` is not used anywhere else, so maybe we could remove it too?

Make sure that it's not used though and also remove a theme for that if my assumption is correct.

Reply:
Ref commit: 4b1a7238